microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -117,7 +122,7 @@ def _install(self) -> bool:
                 isinstance(self.node.os, Ubuntu)
                 and self.node.os.information.version >= "18.4.0"
             )
-            or isinstance(self.node.os, CBLMariner)
+            or isinstance(self.node.os, CBLMariner) or isinstance(self.node.os, SocOs)
```

Comment:
Convert to positive conditions, skip other endorsed distros like RedHat, SUSE, etc.

Reply:
the change is done, thanks

Reply:
Please push latest changes.